### PR TITLE
specifications SIG

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
@open-telemetry/technical-committee
@open-telemetry/specs-approvers 
@danielkhan 
@pavolloffay 

What time will work the best? We probably want to do it early morning pacific time. As we don't have Asia yet. 

I was thinking Thursday or Tuesday 8 AM pacific? Please list your preference

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Tue 8AM PDT works for me

**Comment by AloisReitbauer** [comment]
That would work for me too, Although I would also be fine with a later time.

File: README.md [hunk 1]
```diff
@@ -19,6 +20,21 @@ Meeting invites can be accessed from our public Calendar ([web](https://calendar
...
+of a Technical Committee
+[@open-telemetry/technical-committee](https://github.com/orgs/open-telemetry/teams/technical-committee).
+It is recommended for all members of
+[@open-telemetry/specs-approvers](https://github.com/orgs/open-telemetry/teams/specs-approvers)
+and open for all.
+
+Specifications gitter channel: [![Gitter
+chat](https://badges.gitter.im/open-telemetry/opentelemetry-specification.svg)](https://gitter.im/open-telemetry/opentelemetry-specification?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Meetings: TBD
```
Comment (songy23) [inline README.md]: So the consensus here is Tue 8am? :)

Comment (SergeyKanzhelev) [reply README.md]: yep. Updated

**Comment by SergeyKanzhelev** [comment]
the time seems to be largely accepted. Merging it.

**Pull request merged**